lib/cpp/src/thrift/transport/TServerSocket.cpp

```diff
@@ -16,7 +16,9 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-
+#if defined(_MSC_VER) && (_MSC_VER < 1900)
+#define snprintf _snprintf
```

Comment:
One of our goals early on was to maintain latest version and not try and support everything available as it would become too difficult to maintain. What is the minimum set here that is in LTS, I would say we try and cover latest and possible one or two back (if that)